src/components/autocomplete/demoInsideDialog/dialog.tmpl.html

```diff
@@ -9,7 +10,7 @@
     </div>
   </md-toolbar>
 
-  <md-dialog-content>
+  <md-dialog-content ng-cloak>
```

Comment:
The cloak hides the UI [at initialization] while more complex FlexBox layouts adjust and resize. The toolbar and actions are very simple and IMO do not require the hide trick. But if you want to be consistent, keep all three (3).